weatherman.py

```diff
@@ -0,0 +1,177 @@
+import os
+import sys
+import argparse
+import datetime
+import glob
+import csv
+
+class DayReport:
+  def __str__(self):
+    print(self.date)
+
+  def __init__(self, date, maxtemp, meantemp, mintemp, dew, meandew, mindew, maxhumidity, meanhumidity, minhumidity, maxsealevel, meansealevel, minsealevel, maxvisibility, meanvisibility, minvisibility, maxwind, meanwind, maxgust, precipitaion, cloudcover, events, winddirdegrees):
```

Comment:
A good function have less number of arguments, instead of passing large number of arguments pass only row/weather_record.